samples/outlook/55-display-items/display-existing-appointment.yaml

```
@@ -3,18 +3,29 @@ name: Open an appointment
 description: Displays existing appointment in a separate window
 host: OUTLOOK
 api_set:
-    Mailbox: '1.1'
+    Mailbox: preview
 script:
     content: |
         // Pre-populate with current item ID.
         $("#itemId").val(Office.context.mailbox.item.itemId);
 
         $("#run").click(run);
 
+        $("#run-async").click(runAsync);
+
         function run() {
           var itemId = $("#itemId").val();
           Office.context.mailbox.displayAppointmentForm(itemId);
         }
+
+        // The Async version will return error(9049) if the item is not found
+        // The Async version is only in preview
+        function runAsync() {
+          var itemId = $("#itemId").val();
+          Office.context.mailbox.displayAppointmentFormAsync(itemId, function(asyncResult) {
```

Can you make this change here and in other instances?
```suggestion
        function runAsync() {
          var itemId = $("#itemId").val();

          // The async version will return error 9049 if the item is not found.
          // The async version is only available in preview.
          Office.context.mailbox.displayAppointmentFormAsync(itemId, function(asyncResult) {
```